src/urllib3/response.py

```diff
@@ -460,7 +460,7 @@ def connection(self):
     def isclosed(self):
         return is_fp_closed(self._fp)
 
-    def tell(self):
+    def tell(self) -> int:
```

Comment:
Because this method is called from a typed context, we had to type it, although we haven't addressed the rest of the module yet.